src/datasources/subscription.js

```diff
@@ -29,7 +33,7 @@ export const ingestSubscription = async (params, headers, parsedInfo) => {
   const subscription = new Subscription(headers, requestInfo, params)
 
   // Contact CMR
-  await subscription.ingest(params, ingestKeys, headers)
+  subscription.ingest(params, ingestKeys, headers)
```

Comment:
trying to understand: why does this not need an await since the ingest method is returning a promise? `  ingest(data, requestedKeys, providedHeaders) {
    // eslint-disable-next-line no-param-reassign
    data = mergeParams(data)

    this.logKeyRequest(requestedKeys, 'ingest')

    // Construct the promise that will ingest data into CMR
    this.response = cmrIngest(
      this.getConceptType(),
      data,
      providedHeaders,
      this.ingestPath
    )
  }`

Reply:
We don't need an await because the methods that use its response await, this was the only external call that had an await so I removed it.